tools/validators/instance_validator/validate/generate_universe.py

```diff
@@ -28,6 +28,7 @@
 def BuildUniverse(
     use_simplified_universe: bool = False,
     modified_types_filepath: str = None,
+    default_types_filepath: str = constants.ONTOLOGY_ROOT,
```

Comment:
not sure I follow why we need to pass it here instead of reading it from the constants?

Reply:
because the constants file path won't work for applications that call the instance validator's Run Validation method outside of the instance validator.

For example, take ABEL which runs from `tools\abel\abel.py`. It will call `handler.RunValidation` from the instance validator which will then call `generate_universe.BuildUniverse`. If `BuildUniverse` uses  `constants.ONTOLOGY_ROOT` then this error will arise:

```
Specified filepath [/usr/local/lib/python3.10/dist-packages/validate/../../../../ontology/yaml/resources] for default ontology does not exist
```

This is because the `validate` package is being used but the constant is trying to reference the ontology as if constants exists in the digital buildings repo. `Setup.py` creates these packages. I originally had passed in a different a path from ABEL to the ontology as a modified types path, but then the ontology validator will run and print findings which is what we don't want. My final solution was just to add a default ontology argument so we can explicitly define the default ontology from any application that calls the instance validator instead of relying on the instance validator's constants.

Another solution would be to turn the ontology into a package, or somehow expose the ontology universe as a package with some kind of API.

Reply:
why not update the constants.ONTOLOGY_ROOT?
here you are forcing the user to always set an ontology path?